action/SegmentSurfaces.action

```diff
@@ -1,5 +1,6 @@
 bool save_cloud # True if the point cloud should be saved
 ---
 rapid_pbd_msgs/Landmark[] landmarks
+moveit_msgs/CollisionObject[] surfaces
```

Comment:
This should be a new message type, `rapid_pbd_msgs/Surface surface`:
```
geometry_msgs/PoseStamped pose_stamped # The pose of center of the table. Should be specified in the base frame.
geometry_msgs/Vector3 dimensions # Dimensions of the table, in meters
```

The fact that we are using MoveIt is an internal detail and should not be part of this API.

There should also only be one surface. If we detect multiple surfaces in the future, we can change this.